designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Dtos.scala

```diff
@@ -58,6 +57,11 @@ object Dtos {
           displayableName = "Delete",
           icon = "/assets/activities/actions/delete.svg"
         ),
+        ScenarioActivityActionMetadata(
+          id = "add_comment",
+          displayableName = "Add",
```

Comment:
Will it be used by FE? Is it ok?

Reply:
I think the `Add comment` fits better here. We will display this text on icon hover as a tooltip

Reply:
What about other comment-related actions? Should they have " comment" suffix too?

Reply:
Yes, let's also add this suffix in other cases

Reply:
Ok added. + I imporve date formatting and additional fields for batch